src/libexpr/primops/fetchTree.cc

```diff
@@ -456,6 +456,17 @@ static RegisterPrimOp primop_fetchGit({
           > **Note**
           >
           > This behavior is disabled in *Pure evaluation mode*.
+
+        - To fetch the content of a checked-out work directory:
+
+          ```nix
+          builtins.fetchGit ./work-dir
+          ```
+
+	  If the url points to a local directory, and no `ref` or `rev` is
```

Comment:
```suggestion
	  If the URL points to a local directory, and no `ref` or `rev` is
```

Also, please use Git (the project name) or `git` (the command).